### ISSUE TITLE
latest version of requests is not backwards compatible

### ISSUE THREAD

**Issue by jbrepogmailcom** [psf/requests#5929]
Summary.

## Expected Result
requests(URL, headers, payload) returns payload

What you expected.

## Actual Result
Error, because new version of requests does not accept 3 parameters, only 2. It causes all python programs that use requests with 3 parameters to crash and need to be reprogrammed. 

What happened instead.

## Reproduction Steps

```python
import requests
r = requests.get(URL, headers=headers, params=payload)
```

## System Information

    $ python -m requests.help

```
{
  "chardet": {
    "version": "3.0.4"
  },
  "cryptography": {
    "version": ""
  },
  "idna": {
    "version": "2.8"
  },
  "implementation": {
    "name": "CPython",
    "version": "3.8.10"
  },
  "platform": {
    "release": "5.11.0-27-generic",
    "system": "Linux"
  },
  "pyOpenSSL": {
    "openssl_version": "",
    "version": null
  },
  "requests": {
    "version": "2.22.0"
  },
  "system_ssl": {
    "version": "1010106f"
  },
  "urllib3": {
    "version": "1.25.8"
  },
  "using_pyopenssl": false
}

```

This command is only available on Requests v2.16.4 and greater. Otherwise,
please provide some basic information about your system (Python version,
operating system, &c).

**Comment by nateprewitt** [comment]
Hi @jbrepogmailcom, could you elaborate more on what the actual issue you're hitting is. Nothing changed in v2.x for Requests around this behavior and the example works without issue. I'd recommend checking to see if you have a requests.py file that's overwriting the namespace.

**Comment by jbrepogmailcom** [comment]
Hello, I am very sorry but I corrected the script and I do not have original error message.

What I know for sure is that `r=requests.get(URL, headers=headers, payload=payload)` worked for years until few days (or weeks?) ago. It may have been some automatic linux update that I got recently, but the script stopped working. When I debugged it manually, I was getting error that `r` does not exist on next line. So I ran `r=requests.get(URL, headers, payload)` and I got error message like requests.get accepts only 2 parameters, but not 3.

When I deleted `headers` and changed command to `r=requests.get(URL, payload=payload)` then it worked normally.

(headers and payload were variables)

Jan
